src/lib-voyager.test.ui.tsx

```diff
@@ -377,4 +380,46 @@ describe('lib-voyager', () => {
     }, DEFAULT_TIMEOUT_LENGTH);
   });
 
+  it('getBookmarkedSpecs returns Vega-Lite spec with correct data', done => {
+    setTimeout(() => {
+      try {
+        const spec1 = '{"data":{"name":"source"},"mark":"bar","encoding":{"y":{"field":"Name","type":"nominal",'
+        + '"scale":{"rangeStep":12}}, "x":{"aggregate":"count","field":"*","type":"quantitative",'
+        + '"axis":{"orient":"top"}}}, "config":{"overlay":{"line":true},"scale":{"useUnaggregatedDomain":true}}}';
+        const spec2 = '{"data":{"name":"source"},"mark":"bar","encoding":{"y":{"field":"Cylinders","type":"nominal"},'
+        + '"x":{"aggregate":"count","field":"*","type":"quantitative"}},' +
+        '"config":{"overlay":{"line":true},"scale":{"useUnaggregatedDomain":true}}}';
```

Comment:
Just a tiny nitpick: please tab in the string on line 391 so it is aligned with the strings above it. Thanks!